FRENCH/src/ch13-03-improving-our-io-project.md

```diff
@@ -238,6 +238,18 @@ donné que nous prenons possession de `args` et que nous allons muter `args`
 en itérant dessus, nous pouvons ajouter le mot-clé `mut` dans la spécification
 du paramètre `args` pour le rendre mutable.
 
+<!-- Something is missing here -->
+
+Nous avons aussi besoin de préciser que le type de slice de chaîne de caractères 
+d'erreur ne peut avoir que la durée de vie 'static. Précédemment, nous retournions toujours 
+des littéraux de chaînes de caractères. Cependant, comme nous avions une référence dans les paramètres, 
+il y avait une possibilité que la référence dans le type de retour puisse avoir la même durée de vie que la 
+référence dans les paramètres. Toutefois, comme les règles que nous avons vues dans une section du 
+chapitre 10 s'appliquaient également, nous n'avions pas besoin de préciser la 
+durée de vie de &str. Mais avec le changement que nous venons de faire à args, les règles 
+d'élision de durée de vie ne s'appliquent plus et nous devons désormais indiquer 
+la durée de vie 'static.
+
```

Comment:
```suggestion
```
Looks like both https://github.com/Jimskapt/rust-book-fr/blob/french-release/src/ch13-03-improving-our-io-project.md and https://github.com/rust-lang/book/blob/main/src/ch13-03-improving-our-io-project.md have not this content ?

The compilated book at https://doc.rust-lang.org/book/ch13-03-improving-our-io-project.html have this text, but is clearly old (you can see it on https://doc.rust-lang.org/book/title-page.html that this page has been significally "cleared" recently and Rust version updated)

I have the feeling to have recently removed some big paragraphs like this when updated the book with its source, but I'm not sure this is there, git blame on GitHub does not help quickly, to the moment.

If you disagree I'll take more time to investigate on this 😉

Reply:
I can confirm it was voluntary removed on following commit : https://github.com/Jimskapt/rust-book-fr/commit/12a94a0afada0b630f54bd66480ff12d463ada8e#diff-212536f0dc9379c9c0a1a973f223e1771cbfbc46e8e04442ffe6873fe233c14eL241-L264